assets/js/googlesitekit-data.js

```diff
@@ -6,17 +6,14 @@
  * Internal dependencies
  */
 import SiteKitRegistry from 'assets/js/googlesitekit/data';
```

Comment:
Can we name this `Data` for consistency with elsewhere?

Reply:
Sure thing.